diff -upr a/server/VirtualDrawable.cpp b/server/VirtualDrawable.cpp
--- a/server/VirtualDrawable.cpp	2015-01-16 17:10:59.000000000 -0600
+++ b/server/VirtualDrawable.cpp	2016-03-07 22:32:10.036137613 -0600
@@ -27,7 +27,7 @@ using namespace vglutil;
 using namespace vglserver;
 
 
-#define CHECKGL(m) if(glError()) _throw("Could not "m);
+#define CHECKGL(m) if(glError()) _throw("Could not " m);
 
 // Generic OpenGL error checker (0 = no errors)
 static int glError(void)
