diff -up VirtualGL-2.6.3/server/faker-glx.cpp.mesafix VirtualGL-2.6.3/server/faker-glx.cpp
--- VirtualGL-2.6.3/server/faker-glx.cpp.mesafix	2020-02-28 15:55:05.187424874 -0500
+++ VirtualGL-2.6.3/server/faker-glx.cpp	2020-02-28 16:07:55.049100590 -0500
@@ -2191,12 +2191,21 @@ void glXQueryDrawable(Display *dpy, GLXD
 
 	CATCH();
 }
-
+ 
+#if defined(GLX_GLXEXT_VERSION) && GLX_GLXEXT_VERSION >= 20190000
+void glXQueryGLXPbufferSGIX(Display *dpy, GLXPbuffer pbuf, int attribute,
+       unsigned int *value)
+#else
 int glXQueryGLXPbufferSGIX(Display *dpy, GLXPbuffer pbuf, int attribute,
 	unsigned int *value)
+#endif
 {
 	glXQueryDrawable(dpy, pbuf, attribute, value);
+#if defined(GLX_GLXEXT_VERSION) && GLX_GLXEXT_VERSION >= 20190000
+    return;
+#else
 	return 0;
+#endif
 }
 
 
@@ -2358,8 +2367,13 @@ void glXSwapBuffers(Display *dpy, GLXDra
 
 // Returns the transparent index from the overlay visual on the 2D X server
 
+#if defined(GLX_GLXEXT_VERSION) && GLX_GLXEXT_VERSION >= 20190000
+int glXGetTransparentIndexSUN(Display *dpy, Window overlay, Window underlay,
+       unsigned long *transparentIndex)
+#else
 int glXGetTransparentIndexSUN(Display *dpy, Window overlay, Window underlay,
 	long *transparentIndex)
+#endif
 {
 	int retval = False;
 	XWindowAttributes xwa;
diff -up VirtualGL-2.6.3/server/faker-sym.h.mesafix VirtualGL-2.6.3/server/faker-sym.h
--- VirtualGL-2.6.3/server/faker-sym.h.mesafix	2020-02-28 16:00:16.300060909 -0500
+++ VirtualGL-2.6.3/server/faker-sym.h	2020-02-28 16:01:29.518269213 -0500
@@ -535,9 +535,14 @@ FUNCDEF2(GLXFBConfigSGIX, glXGetFBConfig
 
 // GLX_SUN_get_transparent_index
 
+#if defined(GLX_GLXEXT_VERSION) && GLX_GLXEXT_VERSION >= 20190000
+FUNCDEF4(int, glXGetTransparentIndexSUN, Display *, dpy, Window, overlay,
+       Window, underlay, unsigned long *, transparentIndex,
+       glXGetTransparentIndexSUN)
+#else
 FUNCDEF4(int, glXGetTransparentIndexSUN, Display *, dpy, Window, overlay,
 	Window, underlay, long *, transparentIndex, glXGetTransparentIndexSUN);
-
+#endif
 
 // GL functions
 
